TicTacToe2/TicTacToeGame.cs

```diff
@@ -6,18 +6,21 @@ namespace TicTacToe2
 {
     class TicTacToeGame
     {
-        private readonly string[] space = new string[16] { "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16" };
-        private string player = "X";
+        private readonly char[,] space = new char[1000, 1000];
+        private char player = 'X';
         private int turnCount = 0;
-        private int gridNumber;
-        private int choice = 0;
+
+        private int gridNumber = 0;
+        private int choiceRow = 0;
+        private int choiceColumn = 0;
```

Comment:
Both choiceRow and choiceColumn are used only in GetUserInput() and should be declared inside of that function.

Reply:
I believe I initially, thought this was going to be a bigger deal than it ended up being. I'll try to review my declarations before submitting next time.